slides/index.html

```diff
@@ -20,11 +20,23 @@
 
 ## Hvem er vi?
 
-Studerer datateknikk på NTNU
+???
+
+- Først og fremst takk til fagkom for gratis muffins og kaffe
```

Comment:
"We're giving it away, but it is most certainly not free!"
![image](https://cloud.githubusercontent.com/assets/578029/6433230/b7acd3bc-c069-11e4-9354-955c2bbc7877.png)